# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1324491160 28800
#      Wed Dec 21 10:12:40 2011 -0800
# Branch stable
# Node ID 41511cce048850cd7ffba148e677b5873f6643c8
# Parent  4e8f23ccadcecf3a26188b26272a2466e8411e5f
test: Fix 2nd demo for slice() (bug #35143)

slice.m: Declare variables for 2nd demo (bug #35143)

diff --git a/scripts/plot/slice.m b/scripts/plot/slice.m
--- a/scripts/plot/slice.m
+++ b/scripts/plot/slice.m
@@ -182,12 +182,14 @@ endfunction
 
 
 %!demo
 %! [x, y, z] = meshgrid (linspace (-8, 8, 32));
 %! v = sin (sqrt (x.^2 + y.^2 + z.^2)) ./ (sqrt (x.^2 + y.^2 + z.^2));
 %! slice (x, y, z, v, [], 0, []);
 
 %!demo
+%! [x, y, z] = meshgrid (linspace (-8, 8, 32));
+%! v = sin (sqrt (x.^2 + y.^2 + z.^2)) ./ (sqrt (x.^2 + y.^2 + z.^2));
 %! [xi, yi] = meshgrid (linspace (-7, 7));
 %! zi = xi + yi;
 %! slice (x, y, z, v, xi, yi, zi);
 
